test/shared/connectors/ConnectorSpec.scala

```diff
@@ -152,35 +152,35 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
 
     protected lazy val requiredHeaders: Seq[(String, String)] = requiredDesHeaders
 
-    MockedAppConfig.desBaseUrl returns baseUrl
+    MockedAppConfig.desBaseUrl returns this.baseUrl
```

Comment:
I don't think any of these need the "this.", worth removing?

Reply:
"this" reference is needed here as baseUrl is shadowed and scala compiler throws error without "this."